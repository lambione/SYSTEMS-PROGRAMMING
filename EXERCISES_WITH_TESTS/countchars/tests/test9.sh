#!/bin/sh
exec "$1" fghijklmnopqrstuvwxyz tests/f3 tests/f4
