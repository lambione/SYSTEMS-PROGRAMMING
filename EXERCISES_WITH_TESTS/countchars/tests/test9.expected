tests/f3 f:353 g:401 h:418 i:367 j:384 k:382 l:393 m:382 n:377 o:379 p:372 q:370 r:362 s:383 t:381 u:403 v:379 w:387 x:411 y:379 z:386
tests/f4 f:1 g:6 h:3 i:4 j:3 k:4 l:5 m:6 n:3 o:3 p:4 q:5 r:5 s:1 t:3 u:4 v:2 w:3 x:3 y:4 z:2
